### PR TITLE
Add support for visualizing and updating dependencies (within Gopkg.toml constraints) in dep-enabled projects.

### LINKED ISSUE #81: Support dep
Just like it has an option for govendor and godeps, it should also support [dep](https://github.com/golang/dep) which is now considered production ready.
Thank you.

### PR DESCRIPTION
**Pull request by shurcooL**
This PR adds support for [`dep`](https://github.com/golang/dep) and resolves #81. /cc @sdboyer @bits01 @bradleyfalzon

A new `-dep` flag is added. Using it requires `dep` binary to be available in PATH. When `-dep` flag is set to point to a `Gopkg.toml` file of a `dep`-enabled project, it runs `dep status` in the containing directory to determine available updates (within `Gopkg.toml` constraints) and then displays them in a visual form.

The Update (and Update All) buttons are functional. Pressing them executes `dep ensure -update {root}` command in same directory, causing the update to be applied.

E.g.:

```
$ Go-Package-Store -dep=/Users/Gopher/go/src/github.com/bradleyfalzon/gopherci/Gopkg.toml
Determining the list of Go packages from Gopkg.toml file: /Users/Gopher/go/src/github.com/bradleyfalzon/gopherci/Gopkg.toml
Go Package Store server is running at http://localhost:7043/updates.

dep ensure -update github.com/google/go-github
Done.
```

image

### CODE REVIEW

**Comment by bradleyfalzon** [comment]
This is great, going to give it a test shortly. Did you want to update the README.md with a similar example to the PR's?

**Comment by shurcooL** [comment]
> Did you want to update the README.md with a similar example to the PR's?

Sorry, I don't understand what you mean by that. Can you rephrase?

README.md is already updated to include the new `Go-Package-Store -help` output, which includes a new usage example. Isn't it?

**Comment by bradleyfalzon** [comment]
> which includes a new usage example. Isn't it?

Argh, yes, sorry, I must have skipped over that file when I looked, ignore my comment!

File: updater/dep.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// Dep is an Updater that updates Go packages in a project managed by dep.
+//
+// It requires the dep binary to be available in PATH.
+type Dep struct {
+	// Dir specifies where the dep binary is executed.
+	// If empty, current working directory is used.
+	Dir string
+}
+
+func (d Dep) Update(repo *gps.Repo) error {
```
Comment (shurcooL) [inline updater/dep.go]: Ignoring because this implements an interface, and I don't want to copy/paste the interface documentation into multiple places, causing it to more easily fall out of date.

I could write a `// Update implements Updater interface.`, but it doesn't seem worth it at this point.
